civihr_employee_portal/js/reports.js

```diff
@@ -551,7 +545,9 @@
             CRM.$('.report-config-select').append('<option value="' + data['id'] + '">' + data['label'] + '</option>');
             // Sort options by their labels alphabetically.
             CRM.$('.report-config-select').append(CRM.$('.report-config-select option').remove().sort(function (a, b) {
-              var at = $(a).text(), bt = $(b).text();
+              var at = $(a).text();
+              var bt = $(b).text();
```

Comment:
What does `at` and `bt` mean here?  Can we rename them to more meaningful name?

Reply:
Changed to `aText` and `bText`